pkg/ccl/backupccl/restore_online.go

```diff
@@ -112,6 +118,29 @@ func sendAddRemoteSSTWorker(
 		var batchSize int64
 		const targetBatchSize = 440 << 20
 
+		// TODO(dt): handle tenants
+		prefixes := prefixRewriter{rewrites: make([]prefixRewrite, 0, len(tableRekeys))}
+		for _, i := range tableRekeys {
+
+			if i.OldID == 0 {
+				continue
+			}
+			var desc descpb.Descriptor
+			if err := protoutil.Unmarshal(i.NewDesc, &desc); err != nil {
+				return errors.Wrapf(err, "unmarshalling rekey descriptor for old table id %d", i.OldID)
+			}
```

Comment:
I guess. But we do use other fields of the desc (in regular restore) and I think we will here too once we figure out mvcc-ified writes since we'll need it for incomplete import elision. I suppose we could add some native fields to the proto instead, but as you say -- not for this PR. For now this is just the format for rewrite specs.